Open WorldCat.js

```diff
@@ -494,7 +495,7 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "book",
-				"title": "Newman's relation to modernism",
+				"title": "Newman's relation to modernism.",
```

Comment:
The period is on the [WorldCat website](http://www.worldcat.org/title/newmans-relation-to-modernism/oclc/676747555) and in their RIS export. The point is originally written in the title of the book:

![title-point](https://cloud.githubusercontent.com/assets/5199995/21716592/9aadacac-d40b-11e6-9743-5c7a910dbbd7.png)

Should we still want to change that?